mdb-concept-recommendation.html

```diff
@@ -86,15 +86,26 @@
                 column: {
                     type: Object,
                     value: {}
+                },
+
+                isButtonClicked: {
```

Comment:
This property does not look like something that should be part of component's public api. Prefix property name with underscore(_)

Reply:
Yes its my mistake